projects/demo/src/modules/components/combo-box/examples/5/index.html

```diff
@@ -0,0 +1,21 @@
+<tui-textfield
+    *tuiLet="items$ | async as items"
+    tuiChevron
+>
+    <input
+        #inputRef
+        placeholder="Search country"
+        tuiComboBox
+        [(ngModel)]="value"
+        (input)="search$.next($any($event.target).value)"
```

Comment:
if you enter a value then once the list appears it will not be selected automatically

Reply:
It will be fixed in next PR

Reply:
* https://github.com/taiga-family/taiga-ui/pull/11039